ts/features/idpay/initiative/configuration/xstate/machine.ts

```diff
@@ -123,7 +124,18 @@ const createIDPayInitiativeConfigurationMachine = () =>
                 onDone: {
                   target: "EVALUATING_IBAN_LIST",
                   actions: "loadIbanListSuccess"
-                }
+                },
+                onError: [
+                  {
+                    cond: "isIbanOnlyMode",
+                    actions: ["showIbanLoadingErrorToast", "exitConfiguration"]
```

Comment:
We are showing an error toast to avoid navigation but then we navigate to another screen. It's not a clear behavior imo. 
Maybe an error screen could be a better option?